recipes/trinotate/meta.yaml

```diff
@@ -0,0 +1,45 @@
+about:
+  home: https://trinotate.github.io/
+  license: ''
+  summary: 'Trinotate is a comprehensive annotation suite designed for automatic
+    functional annotation of transcriptomes, particularly de novo assembled
+    transcriptomes, from model or non-model organisms'
+
+build:
+  number: 1
+  skip: True # [osx]
+
+package:
+  name: trinotate
+  version: '3.2.2'
+
+requirements:
+  build:
+      - libgcc
```

Comment:
No problem. I sympathize with having too many items on the to do list. ;-)

I'll try and go back through some of my packages and build them with osx. I may need to pick your brain for that on some of the more complicated compilations, but at least the perl ones should be fairly straightforward.